### ISSUE TITLE
feature requests

### ISSUE THREAD

**Issue by meleu** [fechy/retropie-web-gui#1]
Cool project man! @botolo78 tried to make something similar forking the recalbox-manager and this is the repository: https://github.com/botolo78/RetroPie-Manager

But it seems that he is very busy nowadays and don't have time to maintain it.

If you don't mind, I'll put here the features we have there as a feature request list here.

- Monitor the system health and disk space
- Edit the Emulation Station config file
- Edit the RetroArch config file
- Edit the autostart.sh script
- View the Emulation Station log file
- Manage your BIOS files

**Comment by fechy** [comment]
@meleu just now I've uploaded new version with the first of the requested features:
[ x ] Monitor the system health and disk space

Give it a try, let me know what you think ;)

**Comment by fechy** [comment]
Unfortunately, upon closer look, the CPU usage seems to be a little off. I'll work on it :(

**Comment by meleu** [comment]
As @CourierSS mentioned [on the forum](https://retropie.org.uk/forum/post/30540), the temperature is an important item to monitor.

**Comment by fechy** [comment]
> As @CourierSS mentioned on the forum, the temperature is an important item to monitor.

@meleu agree. I've created an issue for it (#5) and I'll be working on it soon.
